server/db/__mocks__/index.js

```diff
@@ -0,0 +1,11 @@
+
+const addReservation = ({ restaurantId }) => {
+  if (typeof restaurantId === 'number') {
+    // console.log('INSIDE MOCK addReservation OK');
+    return new Promise(resolve => resolve());
```

Comment:
if you dont mind me asking, what does this do?

Reply:
Ah, so this basically simulating a successful database entry.  Since I am using promises for the DB helper functions, my mock function has to do this with a promise as well.  Promises receive a callback with (resolve, reject) as its parameters.  On success, you call `resolve()` and pass in any return values you want to send back, and on failure you call `reject()` with an error message